Makefile

```diff
@@ -0,0 +1,18 @@
+# This is pure a convenience wrapper around calling npm scripts for those
+# with `make` in their veins. Though shalt not add complex logic here.
+
+.PHONY: all
+all:
+	npm ci
```

Comment:
I guess this should be changed after we move away from npm workspaces in #279. I'm going to approve it so you have the choice if this goes before or after.

Reply:
Yup, thanks.